scripts/build_with_EESSI_install.slurm

```diff
@@ -0,0 +1,95 @@
+# these and other scripts of the bot should be made available in some directory or in
+# some directories and the locations should then be configured in some cfg file, e.g., app.cfg
+#   init/minimal_eessi_env --> stored under $SCRIPTS_DIR/init
+#   init/eessi_environment_variables --> stored under $SCRIPTS_DIR/init
+#   init/eessi_softwre_subdir_for_host.py --> stored under $SCRIPTS_DIR/init
+#   configure_easybuild --> stored under $SCRIPTS_DIR
+#   eessi_software_subdir.py --> stored under $SCRIPTS_DIR
+
+
+# setup environment variables for build container
+#   mostly what standard build environment contains with the additional bind mount
+#   for the job directory assumed to be the current directory $(pwd) which is mapped to
+#   /eessi_bot_job inside the container
+export EESSI_TMPDIR=${LOCAL_TMP}/${USER}/EESSI
+export SINGULARITY_CACHEDIR=${EESSI_TMPDIR}/singularity_cache
+export SINGULARITY_BIND="${EESSI_TMPDIR}/var-run-cvmfs:/var/run/cvmfs,${EESSI_TMPDIR}/var-lib-cvmfs:/var/lib/cvmfs,$(pwd):/eessi_bot_job"
+export SINGULARITY_HOME="${EESSI_TMPDIR}/home:/home/${USER}"
+export EESSI_PILOT_READONLY="container:cvmfs2 pilot.eessi-hpc.org /cvmfs_ro/pilot.eessi-hpc.org"
+export EESSI_PILOT_WRITABLE_OVERLAY="container:fuse-overlayfs -o lowerdir=/cvmfs_ro/pilot.eessi-hpc.org -o upperdir=${EESSI_TMPDIR}/overlay-upper -o workdir=${EESSI_TMPDIR}/overlay-work /cvmfs/pilot.eessi-hpc.org"
+
+# create directories needed by the container
+mkdir -p ${EESSI_TMPDIR}/{home,overlay-upper,overlay-work,var-lib-cvmfs,var-run-cvmfs}
+
+# save some environment settings we may need after entering the Gentoo Prefix
+#   softwarelist.yaml could be stored directly by EESSI bot into some job definition file
+echo "export SOFTWARE_FILE=${SOFTWARE_FILE}" > _env
+#   scripts dir could be a directory known through app.cfg or stored in job definition file
+echo "export SCRIPTS_DIR=${SCRIPTS_DIR}" >> _env
+#   only used to set EESSI_TMPDIR, could be taken from a job definition file
+echo "export LOCAL_TMP=${LOCAL_TMP}" >> _env
+#   probably not needed (because all information can be derived via scripts on build node, i.e.,
+#   inside a job); same as "${EESSI_OS_TYPE}/${EESSI_SOFTWARE_SUBDIR}"
+echo "export BUILD_TARGET_ARCHITECTURE=${BUILD_TARGET_ARCHITECTURE}" >> _env
+#   could be delivered as argument, only used to set EESSI_PILOT_VERSION, the latter might be
+#   set directly instead
+echo "export BUILD_EESSI_VERSION=${BUILD_EESSI_VERSION}" >> _env
+
+# the container script launches the container
+#${SCRIPTS_DIR}/launch_build_container.sh ${SCRIPTS_DIR}/run_in_compat_layer_env.sh ${SCRIPTS_DIR}/mod_EESSI_install.sh
+export EESSI_PILOT_VERSION=${BUILD_EESSI_VERSION}
+singularity exec \
+    --bind "${SCRIPTS_DIR}:${SCRIPTS_DIR}" \
+    --fusemount "${EESSI_PILOT_READONLY}" \
+    --fusemount "${EESSI_PILOT_WRITABLE_OVERLAY}" \
+    docker://ghcr.io/eessi/build-node:debian10 \
+    ${SCRIPTS_DIR}/run_in_compat_layer_env.sh ${SCRIPTS_DIR}/mod_EESSI_install.sh
```

Comment:
You should be able to remove this whole block, including defining the related environment variables above, with:

```
export SINGULARITY_BIND="${SCRIPTS_DIR}:${SCRIPTS_DIR}"
./build_container.sh run ${LOCAL_TMP}/${USER}/EESSI ${SCRIPTS_DIR}/mod_EESSI_install.sh
```

That's using the scripts available in https://github.com/EESSI/software-layer/

One small change is needed for `build_container.sh`, since it hard defines `$SINGULARITY_BIND` currently, without taking into account an existing value.

So this line https://github.com/EESSI/software-layer/blob/7ec49a2885cd5e01d808ce6e753711d748f1234a/build_container.sh#L44

should be tweaked to something like:

```shell
# take into account that $SINGULARITY_BIND may be defined already
BIND_PATHS="$EESSI_TMPDIR/var-run-cvmfs:/var/run/cvmfs,$EESSI_TMPDIR/var-lib-cvmfs:/var/lib/cvmfs,$EESSI_TMPDIR"
if [ -z $SINGULARITY_BIND ]; then
    export SINGULARITY_BIND="$BIND_PATHS"
else
    export SINGULARITY_BIND="$SINGULARITY_BIND:$BIND_PATHS"
fi